care/emr/reports/context_builder/data_points/service_request.py

```
@@ -42,6 +44,21 @@
 }
 
 
+class ServiceRequestTagContextBuilder(QuerysetContextBuilder):
+    filterset_class = TagFilter
+    __filterset_backends__ = [filters.DjangoFilterBackend]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)